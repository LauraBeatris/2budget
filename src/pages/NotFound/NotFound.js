import React from "react";
import { Link } from "react-router-dom";
import Header from "../../components/Header/Header";

// Creating the not found page component
const NotFound = () => (
  <div>
    <Header />
    <h1> What are you doing here? </h1>
    <Link to="/dashboard">Go Home</Link>
  </div>
);

export default NotFound;
