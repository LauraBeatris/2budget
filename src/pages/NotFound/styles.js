import styled from "styled-components";

export const Container = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  margin: 20rem 0rem;

  h1 {
    color: rgb(77, 81, 94);
    font-size: 3rem;
  }

  a {
    font-size: 2rem;
    color: #6dbfcc;
  }
`;
