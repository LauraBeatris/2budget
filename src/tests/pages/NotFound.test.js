import { shallow } from "enzyme";
import React from "react";
import NotFound from "../../pages/NotFound/NotFound";

test("should render NotFound correctly", () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
