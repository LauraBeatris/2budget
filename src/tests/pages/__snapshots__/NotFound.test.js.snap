// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound correctly 1`] = `
<div>
  <.sc-htpNat>
    <h1>
       What are you doing here? 
    </h1>
    <Link
      to="/"
    >
      Go Home
    </Link>
  </.sc-htpNat>
</div>
`;
